packages/hdwallet-provider/package.json

```diff
@@ -22,14 +22,14 @@
     "bindings": "^1.5.0",
     "ethereum-cryptography": "^0.1.3",
     "ethereum-protocol": "^1.0.1",
+    "ethereumjs-common": "^1.5.0",
     "ethereumjs-tx": "^2.1.2",
     "ethereumjs-util": "^6.1.0",
     "ethereumjs-wallet": "^1.0.1"
   },
   "devDependencies": {
     "@types/bip39": "^2.4.2",
     "@types/ethereum-protocol": "^1.0.0",
-    "@types/ethereumjs-tx": "^1.0.1",
```

Comment:
Was this unused?

Reply:
It's already listed in `dependencies`.

Reply:
I guess, so - looks like the types are included. If it builds, it should be good, no?